cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -53,21 +53,22 @@ function FilterableListControls( element ) {
     const _expandables = Expandable.init( _dom );
     _expandable = _expandables[0];
 
+    /**
+     * Refresh the height of the filterable list control's expandable
+     * to ensure all its children are visible.
+     */
+    function _refreshExpandableHeight() {
+      window.setTimeout(
+        _expandable.transition.expand.bind( _expandable.transition ),
```

Comment:
I really want to get rid of how this requires the `expand` method call on the transition instead of the expandable. Mental note to update this if that happens.